config/local/generate_toml.go

```diff
@@ -369,6 +370,52 @@ var LocalConfig = config.OracleConfig{
 				},
 			},
 		},
+		{
+			Name: bybit.Name,
+			WebSocket: config.WebSocketConfig{
+				Name:                bybit.Name,
+				Enabled:             true,
+				MaxBufferSize:       1000,
+				ReconnectionTimeout: 10 * time.Second,
+				WSS:                 bybit.ProductionURL,
+				ReadBufferSize:      config.DefaultReadBufferSize,
+				WriteBufferSize:     config.DefaultWriteBufferSize,
+				HandshakeTimeout:    config.DefaultHandshakeTimeout,
+				EnableCompression:   config.DefaultEnableCompression,
+				ReadTimeout:         config.DefaultReadTimeout,
+				WriteTimeout:        config.DefaultWriteTimeout,
+				PingInterval:        20,
```

Comment:
```suggestion
				PingInterval:        20 * time.Second,
```
nit: not sure what unit 20 corresponds to